sys/fuchsia/fidl_process.txt

```diff
@@ -1,13 +1,90 @@
+fuchsia_process_LauncherAddArgsRequest {
+	txid		int32[0]
+	reserved	int32[0]
+	flags		int32[0]
+	ordinal		int32[10]
+	argsInLine	fidl_vector
+	argsOutOfLine	parallel_array[fidl_string, stringnoz]
+} [align_8, packed]
+
+fuchsia_process_LauncherAddEnvironsRequest {
+	txid			int32[0]
+	reserved		int32[0]
+	flags			int32[0]
+	ordinal			int32[11]
+	environInLine		fidl_vector
+	environOutOfLine	parallel_array[fidl_string, stringnoz]
+} [align_8, packed]
+
+fuchsia_process_LauncherAddNamesRequest {
+	txid		int32[0]
+	reserved	int32[0]
+	flags		int32[0]
+	ordinal		int32[12]
+	namesInLine	fidl_vector
+	namesOutOfLine	parallel_array[fuchsia_process_NameInfoInLine, fuchsia_process_NameInfoOutOfLine]
+} [align_8, packed]
+
+fuchsia_process_NameInfoInLine {
+	pathInLine	fidl_string
+	directory	flags[fidl_handle_presence, int32]
+} [align_8, packed]
+
+fuchsia_process_NameInfoOutOfLine {
+	pathOutOfLine	stringnoz
+} [align_8, packed]
+
+fuchsia_process_LauncherAddHandlesRequest {
+	txid			int32[0]
+	reserved		int32[0]
+	flags			int32[0]
+	ordinal			int32[13]
+	handlesInLine		fidl_vector
+	handlesOutOfLine	array[fuchsia_process_HandleInfoInLine]
+} [align_8, packed]
+
+fuchsia_process_HandleInfoInLine {
+	handle	flags[fidl_handle_presence, int32]
+	id	int32
+} [align_8, packed]
+
+zx_channel_call$fuchsia_process_LauncherLaunch(handle zx_chan, options const[0], args ptr[in, fidl_call_args[fuchsia_process_LauncherLaunchRequest, array[int32], array[int32]]], actual_bytes ptr[out, int32], actual_handles ptr[out, int32], read_status ptr[out, int32])
+zx_channel_call$fuchsia_process_LauncherCreateWithoutStarting(handle zx_chan, options const[0], args ptr[in, fidl_call_args[fuchsia_process_LauncherCreateWithoutStartingRequest, array[int32], array[int32]]], actual_bytes ptr[out, int32], actual_handles ptr[out, int32], read_status ptr[out, int32])
+zx_channel_call$fuchsia_process_LauncherAddArgs(handle zx_chan, options const[0], args ptr[in, fidl_call_args[fuchsia_process_LauncherAddArgsRequest, array[int32], array[int32]]], actual_bytes ptr[out, int32], actual_handles ptr[out, int32], read_status ptr[out, int32])
+zx_channel_call$fuchsia_process_LauncherAddEnvirons(handle zx_chan, options const[0], args ptr[in, fidl_call_args[fuchsia_process_LauncherAddEnvironsRequest, array[int32], array[int32]]], actual_bytes ptr[out, int32], actual_handles ptr[out, int32], read_status ptr[out, int32])
+zx_channel_call$fuchsia_process_LauncherAddNames(handle zx_chan, options const[0], args ptr[in, fidl_call_args[fuchsia_process_LauncherAddNamesRequest, array[int32], array[int32]]], actual_bytes ptr[out, int32], actual_handles ptr[out, int32], read_status ptr[out, int32])
+zx_channel_call$fuchsia_process_LauncherAddHandles(handle zx_chan, options const[0], args ptr[in, fidl_call_args[fuchsia_process_LauncherAddHandlesRequest, array[int32], array[int32]]], actual_bytes ptr[out, int32], actual_handles ptr[out, int32], read_status ptr[out, int32])
```

Comment:
This drops all of zx_chan_fuchsia_process_Launcher_client/zx_chan_fuchsia_process_Launcher_server/fdio_service_connect$fuchsia_process_Launcher logic that we did in the previous change. We need all of that generated too.

Reply:
Done.